test/jdk/java/net/Inet4Address/PingThis.java

```diff
@@ -32,6 +32,9 @@
  *          for InetAdress 0.0.0.0 and ::0
  * @run main PingThis
  * @run main/othervm -Djava.net.preferIPv4Stack=true PingThis
+ * @bug 8280498
+ * @summary InetAddress ::0 is not valid, InetAddress 0.0.0.0 only responds
+ *          when an external device reacts to 0.0.0.0
```

Comment:
I think this means the test has two @summary tags, did you mean to add a second?